File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/EdgesBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class EdgesBean {
    private String shape;
    private String relation;
    private String source;
    private String target;
    private String id;
    private List<ControlPointsBean> controlPoints;
    private Integer sourceAnchor;
    private Integer targetAnchor;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/EdgesBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class EdgesBean {
    var shape: String? = null
    var relation: String? = null
    var source: String? = null
    var target: String? = null
    var id: String? = null
    var controlPoints: List<ControlPointsBean>? = null
    var sourceAnchor: Int? = null
    var targetAnchor: Int? = null
}
